### PR TITLE
Implement Card Component

### LINKED ISSUE #48: Implement Card component
Implement the Card React Native component. Could be implemented starting from the following concept:

https://codepen.io/Sevenarth/pen/qBqxdje

### PR DESCRIPTION
**Pull request by amrllkmn**
The PR will close issue #48. Along with Card component, there is also the main Card which displays the IAQ values using SVG.

### CODE REVIEW

**Review by roryort — APPROVED** [review]

File: mobile-app/src/components/AirQuality.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of the HealthyHome project monitoring server
```
Comment (Sevenarth) [inline mobile-app/src/components/AirQuality.js]: monitoring server -> mobile app

File: mobile-app/src/components/ArcGradient.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
+/*
+ * This file is part of the HealthyHome project monitoring server
```
Comment (Sevenarth) [inline mobile-app/src/components/ArcGradient.js]: monitoring server -> mobile app

File: mobile-app/src/components/Card.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of the HealthyHome project monitoring server
```
Comment (Sevenarth) [inline mobile-app/src/components/Card.js]: monitoring server -> mobile app

**Review by Sevenarth — CHANGES_REQUESTED** [review]
Some files are missing the copyright statements too

**Review by Sevenarth — APPROVED** [review]

**Pull request merged**